sdk/eventhub/Azure.Messaging.EventHubs/tests/EventHubConsumerClient/EventHubConsumerClientLiveTests.cs

```diff
@@ -2140,7 +2139,7 @@ await using (var connection = new EventHubConnection(connectionString))
 
                     await using (var invalidProxyConsumer = new EventHubConsumerClient(EventHubConsumerClient.DefaultConsumerGroupName, connectionString, options))
                     {
-                        var readOptions = new ReadEventOptions { MaximumWaitTime = TimeSpan.FromMilliseconds(250) };
+                        var readOptions = new ReadEventOptions { MaximumWaitTime = null };
```

Comment:
Behavior on Linux and macOS is different for this scenario.  They do not immediately terminate with a web socket exception, but continue to attempt to connect until the timeout is exhausted.   The previous setting was causing the test to terminate before the timeout period.   Since it is not important that the read pulls events, setting no wait time and allowing it to block until the timeout period for connections (60 seconds) expires correctly executes on all platforms.

Note that the cancellation token continues to prevent stalling by allowing for twice the connection timeout, just in case.